### PR TITLE
[Feature Fix] Fixed username input displayed in the dropdown sign in form (Firefox) 

### PR DESCRIPTION
**Pull request by revanthkolli**
# Purpose 
#3480 Fixes the display so that the username is in line with the other input/buttons. 

# Changes 
Changed the display: block attribute that is given to the username from the .form-control class to display: inline-block by using a style tag in the html \<input\>. 

# Side Effects 
Checked Safari, Chrome and Firefox and all three seem to be displaying the fields properly now. Not sure about IE.

### CODE REVIEW

**Comment by sloria** [comment]
Adding an inline style doesn't feel like the right fix for this; @revanthkolli have you checked the Bootstrap docs on the proper way to style a form like this? http://getbootstrap.com/css/#forms Perhaps use `form-inline`?

**Comment by revanthkolli** [comment]
@sloria I've tried a couple of different bootstrap classes like form-inline, list-inline and inline but none of them seem to fix the issue.

File: website/templates/nav.mako [hunk 1]
```diff
@@ -73,7 +73,7 @@
             </button>
             <ul class="dropdown-menu" id="menuLogin" role="menu">
               <form class="form" id="signInForm" data-bind="submit: submit" action="${login_url}" method="POST">
-                  <div class="form-group"><input id="email" class="form-control" type="email" data-bind="value: username" name="username" placeholder="Email" aria-label="Username"></div>
+                  <div class="form-group"><input id="email" style="display: inline-block" class="form-control" type="email" data-bind="value: username" name="username" placeholder="Email" aria-label="Username"></div>
```
Comment (lyndsysimon) [inline website/templates/nav.mako]: As we discussed, consider adding the following to `home-page.css` instead:

```css

#signInForm {
    clear:left;
}
```

**Comment by sloria** [comment]
Ok, thanks for looking into this. @caneruguz Are you ok with this fix?

**Comment by caneruguz** [comment]
@revanthkolli the clear fix would make more sense if there were floating elements but as it is I don't see a reason why this error should be happening. It could be a firefox bug as well. 

My suggestion would be, to take a look at layout of the existing system and see if there is anything that can be improved. My inclination is to take a look at btn-group right inside the div that has classes "drowdown sign-in". A btn-group is not necessary in this case and in my tests taking that out resolved layout issue (but introduced other spacing related issues). 

I think it's worthwhile to take a look at this again.

**Comment by lyndsysimon** [comment]
@caneruguz - there is a floating element that's causing an issue - the "Sign In" button. It has a 20px bottom margin that's causing the first item in the form to render on the same line, after it.

I really think clearing the form's left float is the correct solution here.

**Comment by caneruguz** [comment]
@lyndsysimon Okay let's move forward with this fix then.  
@sloria

**Pull request merged**